test/built-ins/Array/prototype/Symbol.iterator/iteration.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2013 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+  description: >
+      The method should return a valid iterator with the context as the
```

Comment:
The description relies on an implicit understanding of the term "valid iterator". Do you think it would be better to describe the behavior of the iterator in the description?